src/BuildScriptGenerator/ManifestFilePropertyKeys.cs

```diff
@@ -20,5 +20,9 @@ public static class ManifestFilePropertyKeys
         internal const string PythonVersion = nameof(PythonVersion);
 
         internal const string HugoVersion = nameof(HugoVersion);
+
+        internal const string PlatformName = nameof(PlatformName);
+
+        internal const string OryxAppType = nameof(OryxAppType);
```

Comment:
nit: remove `Oryx` prefix since this is oryx manifest file...it made sense for environment variable but is not required here...

Reply:
ok